lib/matplotlib/backend_bases.py

```diff
@@ -866,7 +866,7 @@ def get_forced_alpha(self):
         return self._forced_alpha
 
     def get_joinstyle(self):
-        """Return the line join style as one of ('miter', 'round', 'bevel')."""
+        """Return the `.JoinStyle`."""
```

Comment:
For `get_capstyle` you've added "default". Not quite sure which is best "default", "current", blank or something else. Anyway, it should be consistent for `get_capstyle` and `get_joinstyle`.

Reply:
I'll change it back to blank, to be consistent with other properties of the GC.